test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
+{
+public:
+	SolidityExecutionFramework();
+	SolidityExecutionFramework(langutil::EVMVersion _evmVersion);
+
+	Json::Value getMethodIdentifiers()
+	{
+		return m_compiler.methodIdentifiers(m_compiler.lastContractName());
+	}
+
+	dev::bytes compileContract(std::string const& _sourceCode)
+	{
```

Comment:
Please move to .cpp file

Reply:
This will be used by the proto fuzzer as well though, isn't it better to leave it in a common header?

Reply:
I meant move the implementation of the function into the .cpp file. The header file should only contain the simple functions like the constructor in this case. Functions should only be implemented in the header file if they are no longer than a single line (of course there are exceptions).